code/modules/clothing/spacesuits/rig/suits/civilian.dm

```diff
@@ -17,3 +17,33 @@
 		/obj/item/rig_module/healthbar,
 		/obj/item/rig_module/storage/light
 		)
+
+/obj/item/weapon/rig/emergency
+	name = "Emergency RIG"
+	desc = "A light, emergency rig for use by non-qualified personnel in the case of emergency decompression."
+	icon_state = "eva_suit"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 50)
```

Comment:
a spaceproof suit should have some armor. not much, maybe 20 in all fields

Reply:
It is a shitty, emergency rig that only has the purpose of keeping the wearer from suffocating. It replaces the emergency helmet and suit and has the same stats as it's counterpart, the standard emergency space suit.